docs/guides/production.md

```diff
@@ -0,0 +1,296 @@
+*Work In Progress*
+
+There are a million ways to run the `react-server` service in production.  This document is broken into multiple sections to
+describe the elements that go into running your react-server application in production.
+
+# Per-Environment Settings
+
+It helps to setup your project with per-environment settings that are appropriate for your application.  In most cases,
+ things that are helpful in development are not desired in production.  A beta environment may be a mix of development
+ and production settings.  To begin, create a `.reactserverrc` file in the root of your project.  An example of this
+ would be
+ 
+```json
```

Comment:
Good point.  I didn't move it yet though--just trying to get something out there first.  I'm sure people other than me are much better at organizing stuff like this!